alibi_detect/saving/tests/test_validate.py

```diff
@@ -16,19 +16,14 @@
     'x_ref': np.array([[-0.30074928], [1.50240758], [0.43135768], [2.11295779], [0.79684913]]),
     'p_val': 0.05
 }
-cfgs = [mmd_cfg]
-n_tests = len(cfgs)
 
+# Define a detector config dict without meta (as simple as it gets!)
+mmd_cfg_nometa = mmd_cfg.copy()
```

Comment:
Do we need to worry about shallow copies and shared objects here?

Reply:
I did check and it doesn't seem to be an issue. Which is confusing since I think it would be for a list i.e. the pop-ing some from `mmd_cfg_nometa` would have removed it from `mmd_cfg` too 🤷🏻‍♂️ 

I've changed it to `deepcopy` anyway just to be on the safe side...